.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: 'npm' # See documentation for possible values
+    directory: '/' # Location of package manifests
+    schedule:
+      interval: 'monthly'
```

Comment:
Is monthly a better option as weekly?
I am just asking not judging, if monthly is the desired option we should make this change in the other repos too?!

Reply:
I've taken it from our react-native plugin, we can as well move to weekly if you like.

Reply:
Ok, I guess monthly is fine. So its just once a month the work and not every week.